purchase_line_product_image/README.rst

```diff
@@ -0,0 +1,49 @@
+.. image:: https://img.shields.io/badge/licence-LGPL--3-blue.svg
+    :alt: License LGPL-3
+
+Purchase Order Line Product Image for purchases
+==========================================================
```

Comment:
Please align the lenght of the `=` with the length of the header words (like below)